src/helpers/paginate.ts

```diff
@@ -11,23 +11,21 @@ export const perPage = (requestQuery: {
   per_page: string,
   current_page: string
 }) => {
-  const perPage = Number(requestQuery.per_page) || 20
+  const perpage = Number(requestQuery.per_page) || 20
   const currentPage = Number(requestQuery.current_page) || 1
 
   return {
-    perPage,
+    perPage: perpage,
```

Comment:
is this because of the linter too? Still prefer `perPage` instead of `perpage`

Reply:
yes, update here e67523d